test/e2e/ingress.go

```diff
@@ -159,7 +160,13 @@ var _ = framework.KubeDescribe("Loadbalancing: L7 [Feature:Ingress]", func() {
 			// but we want to allow easy testing where a user might've hand
 			// configured firewalls.
 			if framework.ProviderIs("gce", "gke") {
-				ExpectNoError(gcloudCreate("firewall-rules", fmt.Sprintf("ingress-80-443-%v", ns), framework.TestContext.CloudConfig.ProjectID, "--allow", "tcp:80,tcp:443", "--network", "e2e"))
+				// TODO: The KUBE_GCE_NETWORK env var is exported for both
+				// gce/gke e2e test runs, pipe it through just like ProjectID.
+				network := os.Getenv("KUBE_GCE_NETWORK")
```

Comment:
we get this and the gke env var on gke. A flag makes sense, that's why I added the todo. Thoughts on getting this in now and piping a flag in a follow up? want some test coverage over the weekend.